config/src/main/java/tech/pegasys/pantheon/config/GenesisConfigOptions.java

```diff
@@ -28,6 +28,8 @@
 
   boolean isClique();
```

Comment:
Side note: 🤔It would be nice to have a more scaleable way or determining the consensus type than a boolean query for each of the supported types.